README.md

```diff
@@ -12,40 +12,41 @@ Pull requests welcomed.
 
 ### Inital Setup
 Pass in your apikey to the library to authenticate requests:
-```
-var CMDBclass = require("cmdb.js");
-var cmdb = new CMDBclass({
+```js
+const Cmdb = require("cmdb.js");
+const cmdb = new Cmdb({
```

Comment:
Is it confusing for people reading the examples to have 2 constants who name is the same except the case?

Reply:
I don't think so. This is a standard pattern in most OO languages (upper case constructor, lower case instance).

Reply:
👍